plugins/modules/database/misc/redis.py

```diff
@@ -36,6 +36,10 @@
             - The port to connect to
         default: 6379
         type: int
+    login_user:
+        description:
+            - The username to authenticate with
```

Comment:
```suggestion
            - The username to authenticate with.
```
Descriptions should always be proper sentences.